frame/staking/src/lib.rs

```diff
@@ -1474,11 +1474,13 @@ decl_module! {
 			let mut ledger = Self::ledger(&controller).ok_or(Error::<T>::NotController)?;
 
 			let stash_balance = T::Currency::free_balance(&stash);
-
 			if let Some(extra) = stash_balance.checked_sub(&ledger.total) {
 				let extra = extra.min(max_additional);
 				ledger.total += extra;
 				ledger.active += extra;
+				// last check: the new active amount of ledger must be more than ED.
+				ensure!(ledger.active > T::Currency::minimum_balance(), Error::<T>::InsufficientValue);
```

Comment:
```suggestion
				ensure!(ledger.active >= T::Currency::minimum_balance(), Error::<T>::InsufficientValue);
```

can check equality here right?